pkg/client/client.go

```diff
@@ -185,7 +186,7 @@ func (c *Client) SubjectAccessReviews(namespace string) SubjectAccessReviewInter
 	return newSubjectAccessReviews(c, namespace)
 }
 
-// RootSubjectAccessReviews provides a REST client for RootSubjectAccessReviews
+// RootSubjectAccessReviews provides a REST client for SubjectAccessReviews
 func (c *Client) RootSubjectAccessReviews() SubjectAccessReviewInterface {
```

Comment:
We need to settle on terminology.

1. Cluster
2. Server
3. Root??
4. Global??

I prefer cluster because we have a way of identifying a cluster externally with a cluster-id so terminology consistency is there.